doc/manual/integers-and-floating-point-numbers.rst

```diff
@@ -412,14 +412,23 @@ types:
 
 .. doctest::
 
-    julia> (typemin(Float16),typemax(Float16))
-    (-Inf16,Inf16)
```

Comment:
The call sequence that happens is:
```
writemime(STDOUT, MIME"text/plain"(), (Inf16, Inf16)))

writemime(io::IO, ::MIME"text/plain", x) =  showcompact(io, x)
showcompact(io, x) = show(IOContext(STDOUT, :limit_output => true), (Inf16, Inf16))
show(io::IO, t::Tuple) = show_delim_array(io, t, '(', ',', ')', true)
```
`show_delim_array` finally calls `show(io, x)`, but `io = IOContext(STDOUT, :limit_output => true)`
and that calls `showcompact(io, x)`, witch calls `Base.showcompact(io::IO, x::Float16) = _show(io, x, PRECISION, 5, false)`, where the `false` signifies that the number should be printed non-typed.

Now the question is if it is the right behaviour for `showcompact` to drop the type information.